### PR TITLE
Fix bp form by coercing to 0

### PR DESCRIPTION
**Pull request by emord**
@calellowitz @Rohit25negi

### CODE REVIEW

File: custom/icds_reports/utils/aggregation_helpers/delivery_forms.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def aggregation_query(self):
             SUM(CASE WHEN (unscheduled_visit=0 AND days_visit_late < 8) OR
                           (timeend::DATE - next_visit) < 8 THEN 1 ELSE 0 END
                 ) OVER w as valid_visits,
-            where_born
+            LAST_VALUE(where_born) OVER w AS where_born
```
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/delivery_forms.py]: fixes issue where a woman has multiple delivery forms associated with her ccs_record. defaults to the last submitted entry

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def data_from_ucr_query(self):
         LAST_VALUE(anemia) OVER w as anemia,
         LAST_VALUE(anc_abnormalities) OVER w as anc_abnormalities,
         LAST_VALUE(using_ifa) OVER w as using_ifa,
-        LAST_VALUE(ifa_last_seven_days) OVER w as ifa_last_seven_days,
+        MAX(LAST_VALUE(ifa_last_seven_days) OVER w, 0) as ifa_last_seven_days,
```
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: fixes occurrences where ifa_last_seven_days is negative (email sent to app team)

**Review by emord — COMMENTED** [review]
@Rohit25negi these are temporary fixes to ensure no errors. going to let you and Cal figure out the correct workflow later with dashboard and/or app teams

File: custom/icds_reports/utils/aggregation_helpers/delivery_forms.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/delivery_forms.py]: Hey, how can two delivery form be submitted for one mother in one mother?

File: custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py (hunk 2 above)
Comment (Rohit25negi) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: having -ve is valid value or its the worker's mistake

Comment (emord) [reply custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py]: We're not sure. Cal has sent an email to Priyanka and others on team to verify if its an app issue or if its valid

File: custom/icds_reports/utils/aggregation_helpers/delivery_forms.py (hunk 1 above)
Comment (emord) [reply custom/icds_reports/utils/aggregation_helpers/delivery_forms.py]: I'm not sure, but there were multiple occurrences where this was the case

**Comment by emord** [comment]
merging as this is the current state on ICDS and @calellowitz had looked at this while I was debugging last night

**Pull request merged**